src/Tring/OutputPrinter.cs

```diff
@@ -152,20 +169,20 @@ private static void PrintRequest(ConnectionResult request)
                     Console.ForegroundColor = ConsoleColor.Red;
                     break;
             }
-            Console.Write($"{ipOrError.PadRight(15)}");
+            Console.Write($"{ipOrError.PadRight(_lenghtIP)}");
             Console.ResetColor();
-            Console.Write($" | {request.Request.Port.ToString().PadRight(5)} | ");
```

Comment:
I would prefer the usage of [string interpolation alignment specifiers](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) iso `.PadRight`.